doc/topics/releases/sodium.rst

```diff
@@ -0,0 +1,28 @@
+:orphan:
+
+====================================
+Salt Release Notes - Codename Sodium
+====================================
+
+
+Salt mine updates
+=================
+
+Syntax update
+-------------
+
+The syntax for defining salt functions in config or pillar files has changed to
```

Comment:
Also needs updating

Reply:
Updated in bfee1c8d9bcab7685bbc95f5d05a493db52560e9